multiple_listeners/test_on_the_fly.py

```diff
@@ -108,9 +156,7 @@ def test_change_config_on_the_fly(self):
             )
 
         # change config and reload Tempesta
-        tempesta.config.defconfig = make_tempesta_config(
-            SOCKET_AFTER_RELOAD,
-        )['config']
+        tempesta.config.defconfig = self.tempesta_after_reload['config']
         tempesta.reload()
```

Comment:
I still have the issue with bad command and hung test on the last version and #1643 branch:
```
----------------------------------------------------------------------
Running functional tests...
----------------------------------------------------------------------

	Run command 'echo "Start test: multiple_listeners.test_mixed.TestMixedListeners.test_mixed_h2_success" > /dev/kmsg' on host localhost with environment {}
test_mixed_h2_success (multiple_listeners.test_mixed.TestMixedListeners) ... 	Init test case...
	Run command 'date +%s.%N' on host localhost with environment {}
	Starting Nginx on nginx
	Run command 'netstat --inet -apn' on host localhost with environment {}
	Run command 'nginx -c /tmp/nginx/nginx_nginx.cfg' on host localhost with environment {}
	Teardown
Not started
Not started
Not started
Not started
Not started
	Stopping Nginx on nginx
	Run command '[ -e '/tmp/nginx/nginx_nginx.pid' ] && pid=$(cat /tmp/nginx/nginx_nginx.pid) && kill -s TERM $pid && while [ -e '/proc/$pid' ]; do sleep 1; done' on host localhost with environment {}
	Removing Nginx config for nginx
Not started
Removing interfaces
	Run command 'journalctl -k -o cat --since=@1661431961.846103' on host localhost with environment {}
ERROR
	Run command 'echo "End test:   multiple_listeners.test_mixed.TestMixedListeners.test_mixed_h2_success" > /dev/kmsg' on host localhost with environment {}
	Run command 'echo "Start test: multiple_listeners.test_mixed.TestMixedListeners.test_mixed_https_success" > /dev/kmsg' on host localhost with environment {}
test_mixed_https_success (multiple_listeners.test_mixed.TestMixedListeners) ... 	Init test case...
	Run command 'date +%s.%N' on host localhost with environment {}
	Starting Nginx on nginx
	Run command 'netstat --inet -apn' on host localhost with environment {}
	Run command 'nginx -c /tmp/nginx/nginx_nginx.cfg' on host localhost with environment {}
	Run command 'date +%s.%N' on host localhost with environment {}
	Run command 'sysctl net.core.message_cost' on host localhost with environment {}
	Run command 'sysctl -w net.core.message_cost=0' on host localhost with environment {}
	Starting TempestaFW on localhost
	Run command '/root/tempesta/scripts/tempesta.sh --start' on host localhost with environment {'TFW_CFG_PATH': '/tmp/tempesta/tempesta.conf'}
	Run command 'journalctl -k -o cat --since=@1661431964.504066' on host localhost with environment {}
	Run command 'journalctl -k -o cat --since=@1661431964.504066' on host localhost with environment {}
	Run command 'sysctl -w net.core.message_cost=5
' on host localhost with environment {}
Running client
	Run command 'curl -Ikf --http1.1 https://127.0.0.1:4433/' on host localhost with environment {}
	Client exit
Stopping client
Client is stopped
Running client
	Run command 'curl -Ikf --http1.1 https://127.0.0.1:443/' on host localhost with environment {}
Process Process-2:
Traceback (most recent call last):
  File "/root/tempesta-test/helpers/remote.py", line 87, in run_cmd
    assert p.returncode == 0, \
AssertionError: Cmd: 'curl -Ikf --http1.1 https://127.0.0.1:443/' return code is not 0 (35).

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.10/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/lib/python3.10/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/root/tempesta-test/framework/client.py", line 9, in _run_client
    res = remote.client.run_cmd(client.cmd, timeout=(client.duration + 5))
  File "/root/tempesta-test/helpers/remote.py", line 93, in run_cmd
    raise Exception("Remote error: %s, stdout = %s, stderr = %s" %
Exception: Remote error: Error running command 'curl -Ikf --http1.1 https://127.0.0.1:443/' on localhost, stdout = b'', stderr = b'  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current\n                                 Dload  Upload   Total   Spent    Left  Speed\n\r  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0\r  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0\ncurl: (35) error:0A000460:SSL routines::reason(1120)\n'
^C^C	Run command 'echo "End test:   multiple_listeners.test_mixed.TestMixedListeners.test_mixed_https_success" > /dev/kmsg' on host localhost with environment {}
Traceback (most recent call last):
  File "/root/tempesta-test/./run_tests.py", line 322, in <module>
    result = testRunner.run(testsuite)
  File "/usr/lib/python3.10/unittest/runner.py", line 184, in run
    test(result)
  File "/usr/lib/python3.10/unittest/suite.py", line 84, in __call__
    return self.run(*args, **kwds)
  File "/usr/lib/python3.10/unittest/suite.py", line 122, in run
    test(result)
  File "/usr/lib/python3.10/unittest/case.py", line 650, in __call__
    return self.run(*args, **kwds)
  File "/usr/lib/python3.10/unittest/case.py", line 591, in run
    self._callTestMethod(testMethod)
  File "/usr/lib/python3.10/unittest/case.py", line 549, in _callTestMethod
    method()
  File "/root/tempesta-test/multiple_listeners/test_mixed.py", line 216, in test_mixed_https_success
    self.check_curl_response(self.make_curl_request('curl-https-false'), fail=True)
  File "/root/tempesta-test/multiple_listeners/test_mixed.py", line 162, in make_curl_request
    self.wait_while_busy(client)
  File "/root/tempesta-test/framework/tester.py", line 351, in wait_while_busy
    item.wait_for_finish()
  File "/root/tempesta-test/framework/client.py", line 159, in wait_for_finish
    while self.is_busy(verbose=False):
  File "/root/tempesta-test/framework/client.py", line 85, in is_busy
    busy = not self.exit_event.is_set()
  File "/usr/lib/python3.10/multiprocessing/synchronize.py", line 328, in is_set
    with self._cond:
  File "/usr/lib/python3.10/multiprocessing/synchronize.py", line 233, in __exit__
    return self._lock.__exit__(*args)
  File "/usr/lib/python3.10/multiprocessing/synchronize.py", line 98, in __exit__
    return self._semlock.__exit__(*args)
  File "/usr/lib/python3.10/unittest/signals.py", line 36, in __call__
    self.default_handler(signum, frame)
KeyboardInterrupt

```